index.html

```diff
@@ -22,7 +22,7 @@
     body {
       margin: 0;
       font-family: 'Ubuntu', sans-serif;
-      font-weight: 300;
+      font-weight: normal;
```

Comment:
Working, though this looks like it isn't meant to be included. It increases the weight on Firefox.

Otherwise, I am happy.